scripts/irods/test/test_iquest.py

```diff
@@ -108,14 +108,15 @@ def test_iquest_resc_hier_with_like__3714(self):
         self.admin.assert_icommand("irm -f " + filename)
         remove_large_hierarchy(self, LEAF_COUNT, directory)
 
+    # TODO This test can be removed now that iquest relies on a real parser for GenQuery1.
     def test_iquest_matches_names_containing_apostrophes_via_the_equals_operator__issue_4887(self):
         data_object = "data' object"
         self.admin.assert_icommand(['istream', 'write', data_object], input='hello, world')
-        self.admin.assert_icommand(['iquest', "select DATA_NAME where DATA_NAME = '{0}'".format(data_object)], 'STDOUT', ['DATA_NAME = ' + data_object])
+        self.admin.assert_icommand(['iquest', "select DATA_NAME where DATA_NAME = '{0}'".format(data_object.replace("'", '\\x27'))], 'STDOUT', ['DATA_NAME = ' + data_object])
```

Comment:
Notice the use of `.replace`. This is a breaking change depending on how you view things.